Patch to Fix FTBFS due to invalid const char * conversion (#548521), Thanks to
Ilya Barygin <barygin@gmail.com>
Index: speech-tools-1.2.96~beta/speech_class/EST_wave_io.cc
===================================================================
--- speech-tools-1.2.96~beta.orig/speech_class/EST_wave_io.cc	2009-09-26 12:20:49.000000000 +0400
+++ speech-tools-1.2.96~beta/speech_class/EST_wave_io.cc	2009-09-26 12:21:25.000000000 +0400
@@ -68,7 +68,7 @@

 int nist_get_param_int(const char *hdr, const char *field, int def_val)
 {
-    char *p;
+    const char *p;
     int val;

     if (((p=strstr(hdr,field)) != NULL) &&
@@ -84,7 +84,8 @@

 char *nist_get_param_str(const char *hdr, const char *field, const char *def_val)
 {
-    char *p,*val;
+    const char *p;
+    char *val;
     int size;

     if (((p=strstr(hdr,field)) != NULL) &&
